cli/lib/kontena/client.rb

```diff
@@ -55,6 +55,7 @@ def initialize(api_url, token = nil, options = {})
         connect_timeout: ENV["EXCON_CONNECT_TIMEOUT"] ? ENV["EXCON_CONNECT_TIMEOUT"].to_i : 5,
         read_timeout:    ENV["EXCON_READ_TIMEOUT"]    ? ENV["EXCON_READ_TIMEOUT"].to_i    : 30,
         write_timeout:   ENV["EXCON_WRITE_TIMEOUT"]   ? ENV["EXCON_WRITE_TIMEOUT"].to_i   : 5,
+        ssl_ca_path:     ENV["EXCON_SSL_CA_PATH"]     ? ENV["EXCON_SSL_CA_PATH"] : "/etc/ssl/certs",
```

Comment:
Maybe you can add that to https://github.com/kontena/kontena/blob/master/cli/lib/kontena/command.rb#L213 in this PR